cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/Handler.kt

```diff
@@ -166,6 +167,13 @@ abstract class Handler<ResultNode : Node?, HandlerNode, L : LanguageFrontend<in
         return null
     }
 
+    /** Creates a random name starting with a prefix plus a random UUID (version 4). */
+    fun getRandomTempName(): Name {
+        val prefix = "TEMP-RANDOM-NAME"
```

Comment:
After some thought I would like to propose three things:
- probably move this to a companion object of `Name`
- move the prefix as a parameter (maybe with a default)
- get rid of `-` (or at least have `_`), we should make this "name" as natural as possible, so that we don't create any future problems if we ever try to parse this as a real name in the language or something like that. most languages won't accept `-` in the var name I guess?

```
var myLoopVar = Name.random("loopVar")
```

and then I end up with `loopVar1231938190238012301023213`

Reply:
Thoughts about the current version?

Reply:
Looks good! How about just `random` instead of the very long `getRandomTempName` ?

Reply:
I renamed it. However, I'm not a fan of having super short names -> I like names that describe things. You can just hit TAB in your idea and don't have any additional typing work to do...